buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/BuildCommitDistribution.kt

```diff
@@ -75,7 +75,7 @@ open class BuildCommitDistribution : DefaultTask() {
             "--init-script",
             File(checkoutDir, "gradle/init-scripts/build-scan.init.gradle.kts").absolutePath,
             "clean",
-            ":distributions:install",
+            "install",
```

Comment:
Has it been fixed that `install` also installs stuff into the local `.m2` directory?

Reply:
Sorry I am not sure what you are talking about. Can you point me to some description of what is broken/what used to work?

Reply:
Nothing seems to be broken. I remember from some time ago that running `install` without specifying a project caused running the `install` task from the `maven` plugin. I suppose now that we don't apply the `maven` plugin any more, there is also no `install` task to run for every project, right?

I think it still would prefer to qualify which `install` task to run here, so to use `:distributions:install`. Why did you change the invocation to run just `install`?

Reply:
Got it. That problem does not exist anymore since we moved to the "maven-publush" plugin. (I wasn't aware myself that we had it before, but it makes sense what you say.)

So locally you can now safely run `install`, which is nice as the full qualified name now became longer with recent changes.

I changed it here because it now became `distributionFull:install`. And because this builds the master version of Gradle, the change can only be made after the merge, if you want CI to pass on the branch.

But now that this is merged, I can change it to the fully qualified name again for clarity on master directly.

Reply:
Got it. Thank you for the explanation.

Reply:
https://github.com/gradle/gradle/commit/3ef0726b2a184e8fdb493f4bc819cc395cf86451